### ISSUE TITLE
There is no status update of the applicant, even though I update the delivery system

### ISSUE THREAD

**Issue by illumkhan** [devansingaram/tribehired#182]
After updating the delivery system for nicholus, I should be able to see that information for nicholus but I dont. Need to link it to the delivery system
![image](https://f.cloud.github.com/assets/2082445/328038/6f07e436-9b8a-11e2-9503-2205be7762e5.png)

**Comment by devansingaram** [comment]
yes, for now we only show candidate applied, we need to add other steps in soon
